Scripts/build.swift

```diff
@@ -20,38 +20,132 @@ enum TaskError: Error {
     case code(Int32)
 }
 
-guard CommandLine.arguments.count > 3 else {
-    print("Usage: build.swift sdk destination should_test? enable_code_coverage?")
+enum Platform: String, CaseIterable, CustomStringConvertible {
+    case iOS_12
+    case iOS_13
+    case tvOS_12
+    case tvOS_13
+    case macOS_10_15
+    case watchOS_5
+    case watchOS_6
+
+    var destination: String {
+        switch self {
+        case .iOS_12:
```

Comment:
Just curious, why did you decide to use iPad instead of iPhones?

Reply:
CI machines tend to have multiple OS installs for a particular iPad, but less so for particular iPhones.